src/components/sections/Gallery.css

```
@@ -51,132 +77,196 @@
-  width: 12px;
-  height: 12px;
+  min-width: 25px;
+  min-height: 25px;
+  top: -10px;
+  position: relative;
   border-radius: 50%;
   border: none;
   background: #e9ecef;
   cursor: pointer;
   transition: all 0.3s ease;
+  border: 2px solid transparent;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)